build/NuSpecs/build-nupkg.ps1

```diff
@@ -0,0 +1,164 @@
+
+if (!$BuildOutput)
+{
+    Write-Host "Must specify BuildOutput parameter" -ForegroundColor Red
+    Exit 1
+}
+
+if ($VersionOverride)
+{
+    $version = $VersionOverride
+}
+else
+{
+    [xml]$customProps = (Get-Content ..\..\version.props)
+    $versionMajor = $customProps.GetElementsByTagName("ProjectReunionVersionMajor").'#text'
+    $versionMinor = $customProps.GetElementsByTagName("ProjectReunionVersionMinor").'#text'
+    $versionPatch = $customProps.GetElementsByTagName("ProjectReunionVersionPatch").'#text'
+
+    if ((!$versionMajor) -or (!$versionMinor) -or (!$versionPatch))
+    {
+        Write-Error "Expected ProjectReunionVersionMajor, ProjectReunionVersionMinor, and ProjectReunionVersionPatch tags to be in version.props file"
+        Exit 1
+    }
+
+    $version = "$versionMajor.$versionMinor.$versionPatch"
+
+    Write-Verbose "Version = $version"
+}
+
+if ($prereleaseversion)
+{
+    $versiondate = $DateOverride
+    if (-not $versiondate)
+    {
+        $pstZone = [System.TimeZoneInfo]::FindSystemTimeZoneById("Pacific Standard Time")
+        $pstTime = [System.TimeZoneInfo]::ConvertTimeFromUtc((Get-Date).ToUniversalTime(), $pstZone)
+        $versiondate += ($pstTime).ToString("yyMMdd")
+    }
+
+    $version = "$version-$prereleaseversion.$versiondate$subversion"
+   
+}
+
+if (!(Test-Path $OutputDir)) { mkdir $OutputDir }
+
+$nupkgtitle = "Microsoft.ProjectReunion"
+
+function New-TemporaryDirectory {
+    $parent = [System.IO.Path]::GetTempPath()
+    $name = [System.IO.Path]::GetRandomFileName()
+    New-Item -ItemType Directory -Path (Join-Path $parent $name)
+}
+
+function Copy-IntoNewDirectory {
+    Param($source, $destinationDir, [switch]$IfExists = $false)
+
+    if ((-not $IfExists) -or (Test-Path $source))
+    {
+        Write-Verbose "Copy from '$source' to '$destinationDir'"
+        if (-not (Test-Path $destinationDir))
+        {
+            $ignore = New-Item -ItemType Directory $destinationDir
+        }
+        Copy-Item -Recurse -Force $source $destinationDir
+    }
+}
+
+$TempDir = New-TemporaryDirectory
+
+Write-Verbose "TempDir = $($TempDir.FullName)"
+
+# nuspecs can't have conditionals so copy the contents of the runtime folder over, skipping arm64 if those sources aren't
+# created in this particular build.
+$runtimesDir = "$($TempDir.FullName)\runtimes"
+$toolsDir = "$($TempDir.FullName)\tools"
+
+Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll "$runtimesDir\win10-x86\native"
+Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.pri "$runtimesDir\win10-x86\native"
+Copy-IntoNewDirectory -IfExists ..\..\dev\Materials\Acrylic\Assets\NoiseAsset_256x256_PNG.png "$runtimesDir\win10-x86\native\Microsoft.ProjectReunion\Assets"
+Copy-IntoNewDirectory -IfExists $BuildOutput\$BuildFlavor\x64\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll "$runtimesDir\win10-x64\native"
```

Comment:
remove

Reply:
remove lines 103-106?  Is this duplicative?

Reply:
Remove the packaging of that png asset -- it won't be present in Reunion.

Reply:
(So lines 105, 108, 11, 114.)

Reply:
thanks, fixing.